scan/context.go

```diff
@@ -170,12 +174,14 @@ func (s *Scanner) getContextFromRegistry(ctx context.Context, registryArtifact s
 // - The response is 5xx
 func getWithStatusError(url string) (resp *http.Response, err error) {
 	attempt := 0
-
 	for attempt < maxRetries {
 		resp, err = http.Get(url)
 		if err != nil {
+			time.Sleep(util.GetExponentialBackoff(attempt))
+			attempt++
 			continue
```

Comment:
It looks missing `attemp++` here? if `http.Get(url)` has `err`, then it will run an infinity loop.

Reply:
good catch